go.mod

```diff
@@ -6,23 +6,29 @@ go 1.20
 
 require (
 	github.com/alecthomas/participle/v2 v2.0.0
+	github.com/cockroachdb/errors v1.11.3
```

Comment:
i don't think it makes sense to source the cockroachdb errors package for a test. Let's revert the go.mod/go.sum changes.

Reply:
This is a utility library which has useful function to construct meaningful error. So good to keep it

Reply:
how is it different than just using `fmt.Errorf` instead? I agree with @jacques-n that I don't see the benefit in the cockroachdb errors package. What functionality is it providing that we can't already get from `errors.New` and `fmt.Errorf`?

Reply:
I felt instead of using some methods from fmt and some from standard "errors". It is good to use one library. This cockroach library has much richer APIs (e.g. adding Hint to errors) too. Hence I used it so that we can use a single lib for errors and augment errors to be more informative
Though, for this task I could use existing fmt.Errorf. So I reverted back to fmt instead of using cockroach/errors. I still feel this API will be useful to report more meaningful error. I will reintroduce in future when I use APIs of this lib which aren't available in standard package